test/simulator/timestamp/call_log_manager/call_log_manager.h

```diff
@@ -29,7 +29,10 @@
 #pragma once
 
 #include "connection_simulator.h"
+#pragma GCC diagnostic push
```

Comment:
Can we add a comment to explain why this is required?

Reply:
I've removed this since it should no longer be required.